src/test/java/org/folio/services/domainevent/CommonDomainEventPublisherTest.java

```diff
@@ -40,7 +43,7 @@
   @Before
   public void setUpPublisher() {
     eventPublisher = new CommonDomainEventPublisher<>(
-      Map.of(), INVENTORY_INSTANCE, producerManager, failureHandler);
+      Map.of(), KafkaTopic.instance("", environmentName()), producerManager, failureHandler);
```

Comment:
Topic without tenant id on purpose?

Reply:
I will add a tenant here. 

It doesn't seem to make any difference to the test (which I'm a little surprised by given it's test for publishing, probably related to mocking).